examples/lighting-app/beken/main/chipinterface.cpp

```diff
@@ -33,15 +33,51 @@
 #include <setup_payload/ManualSetupPayloadGenerator.h>
 #include <setup_payload/QRCodeSetupPayloadGenerator.h>
 
+//#if CONFIG_ENABLE_OTA_REQUESTOR
+#include "app/clusters/ota-requestor/BDXDownloader.h"
+#include "app/clusters/ota-requestor/OTARequestor.h"
+#include "platform/Beken/OTAImageProcessorImpl.h"
+#include "platform/GenericOTARequestorDriver.h"
+//#endif
+
+using chip::OTAImageProcessorImpl;
+using chip::BDXDownloader;
+using chip::ByteSpan;
+using chip::EndpointId;
+using chip::FabricIndex;
+using chip::GetRequestorInstance;
+using chip::NodeId;
+using chip::OnDeviceConnected;
+using chip::OnDeviceConnectionFailure;
+using chip::OTADownloader;
+using chip::OTAImageProcessorParams;
+using chip::OTARequestor;
+using chip::PeerId;
```

Comment:
Please check whether we really need all using namespace. I think PeerId is unnecessary